test/jdk/com/sun/crypto/provider/Cipher/AEAD/GCMParameterSpecTest.java

```diff
@@ -36,7 +42,13 @@
  */
 public class GCMParameterSpecTest {
 
-    private static final int[] IV_LENGTHS = { 96, 8, 1024 };
+    /*
+     * OpenSSL3 only supports IV lengths up to 16 bytes.
+     * When the IV length is set to be larger than 16Bytes, an error is thrown.
+     * According to the OpenSSL docs(https://www.openssl.org/docs/man1.1.1/man3/EVP_CIPHER_CTX_block_size.html),
+        in OpenSSL1.1.1 and older, there is no error thrown but unpredictable behavior will happen for large IV sizes.
+     */
```

Comment:
You only fixed one of the occurrences of `16Bytes`, and each continuation line of the comment should start with `*`; I suggest:
```
    /*
     * OpenSSL3 only supports IV lengths up to 16 bytes.
     * When the IV length is set to be larger than 16 bytes, an error is thrown.
     * According to the OpenSSL docs([1]), in OpenSSL1.1.1 and older, there is
     * no error thrown but unpredictable behavior will happen for large IV sizes.
     *
     * [1] https://www.openssl.org/docs/man1.1.1/man3/EVP_CIPHER_CTX_block_size.html
     */
```

Reply:
Modified as suggested.